API.md

```diff
@@ -254,6 +281,18 @@ Takes a function which accepts a node-style callback and returns a new function
 const upload = promisify(s3.upload, s3)
 ```
 
+### setProp
+
+```haskell
+setProp : String -> { k: v } -> a -> { k: v }
+```
+
+Accepts a key, object and any value.  Sets the property `key` on the object to `value`.
+
+```js
+setProp('foo', {}, 'bar') //=> { foo: 'bar' }
```

Comment:
Is this the same as ramda's `assoc` but in a different argument order? What is a use case where we would not be passing the data structure in last?